src/frontend/apps/desk/src/app/page.tsx

```diff
@@ -9,11 +9,15 @@ import { Teams } from './Teams';
 import styles from './page.module.css';
 
 export default function Home() {
-  const { initAuth, authenticated } = useAuthStore();
+  const { initAuth, authenticated, initialized } = useAuthStore();
 
   useEffect(() => {
+    if (initialized) {
+      return;
+    }
+
     initAuth();
```

Comment:
It is because of that:
https://github.com/numerique-gouv/people/blob/a7bf530509522dc6c291facbed882c76845c6db0/src/frontend/apps/desk/src/api/fetchWrapper.ts#L36

The token has an expiration (after 2mn I think), and we get a 401, so I logout our user, when I am logout I want `initAuth` to be trigger again, it is like that that we have our token rotation for the moment.